src/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/table/table.js

```diff
@@ -0,0 +1,175 @@
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiSearchBar,
+  EuiBasicTable,
+  EuiButton,
+  EuiIcon,
+  EuiLink,
+  EuiSpacer,
+} from '@elastic/eui';
+import { getSavedObjectIcon } from '../../../../lib/get_saved_object_icon';
+
+export class Table extends PureComponent {
+  static propTypes = {
+    selectedSavedObjects: PropTypes.array.isRequired,
+    selectionConfig: PropTypes.shape({
+      itemId: PropTypes.string.isRequired,
+      selectable: PropTypes.func,
+      selectableMessage: PropTypes.func,
+      onSelectionChange: PropTypes.func.isRequired,
+    }).isRequired,
+    filterOptions: PropTypes.array.isRequired,
+    onDelete: PropTypes.func.isRequired,
+    onExport: PropTypes.func.isRequired,
+    getEditUrl: PropTypes.func.isRequired,
+    goInApp: PropTypes.func.isRequired,
+
+    pageIndex: PropTypes.number.isRequired,
+    pageSize: PropTypes.number.isRequired,
+    items: PropTypes.array.isRequired,
+    totalItemCount: PropTypes.number.isRequired,
+    onQueryChange: PropTypes.func.isRequired,
+    onTableChange: PropTypes.func.isRequired,
+    isSearching: PropTypes.bool.isRequired,
+
+    onShowRelationships: PropTypes.func.isRequired,
+  };
+
+  render() {
+    const {
+      pageIndex,
+      pageSize,
+      items,
+      totalItemCount,
+      isSearching,
+      filterOptions,
+      selectionConfig: selection,
+      onDelete,
+      onExport,
+      selectedSavedObjects,
+      onQueryChange,
+      onTableChange,
+      goInApp,
+      getEditUrl,
+      onShowRelationships,
+    } = this.props;
+
+    const pagination = {
+      pageIndex: pageIndex,
+      pageSize: pageSize,
+      totalItemCount: totalItemCount,
+      pageSizeOptions: [5, 10, 20, 50],
+    };
+
+    const filters = [
+      {
+        type: 'field_value_selection',
+        field: 'type',
+        name: 'Type',
+        multiSelect: 'or',
+        options: filterOptions,
+      },
+      // Add this back in once we have tag support
+      // {
+      //   type: 'field_value_selection',
+      //   field: 'tag',
+      //   name: 'Tags',
+      //   multiSelect: 'or',
+      //   options: [],
+      // },
+    ];
+
+    const columns = [
+      {
+        field: 'type',
+        name: 'Type',
+        width: '35px',
```

Comment:
`width` should be a little bigger, Type is nearly touching the checkbox:

<img width="108" alt="screen shot 2018-04-03 at 2 59 02 pm" src="https://user-images.githubusercontent.com/1965714/38278126-a3021912-374f-11e8-885e-c4d1ba921c5b.png">

Reply:
Sure!